build/ltmain.sh

```diff
@@ -3467,7 +3467,7 @@ EOF
 	tempremovelist=`$echo "$output_objdir/*"`
 	for p in $tempremovelist; do
 	  case $p in
-	    *.$objext)
+	    *.$objext|*.gcno)
```

Comment:
```suggestion
	    *.$objext | *.gcno)
```
 
This is just to have this diff as much similar to the upstream as possible.